apps/dash-salesforce-crm/index.py

```diff
@@ -0,0 +1,126 @@
+from panels import opportunities, cases, leads
+
+app.layout = html.Div(
+    [
+        html.Div(
+            className="row header",
+            children=[
+                html.Button(id="menu", children=dcc.Markdown("&#8801")),
+                html.Span(
+                    className="app-title",
+                    children=[
+                        dcc.Markdown("**CRM App**"),
+                        html.Span(
+                            id="subtitle",
+                            children=dcc.Markdown("&nbsp using Salesforce API"),
+                            style={"font-size": "1.8rem", "margin-top": "15px"},
+                        ),
+                    ],
+                ),
+                html.Img(src=app.get_asset_url("logo.png")),
+                html.A(
+                    id="learn_more",
+                    children=html.Button("Learn More"),
+                    href="https://plot.ly/dash/",
+                ),
+            ],
+        ),
+        html.Div(
+            id="tabs",
+            className="row tabs",
+            children=[
+                dcc.Link(dcc.Markdown("Opportunities"), href="/opportunities"),
+                dcc.Link(dcc.Markdown("Leads"), href="/leads"),
+                dcc.Link(dcc.Markdown("Cases"), href="/cases"),
+            ],
+        ),
+        html.Div(
+            id="mobile_tabs",
+            className="row tabs",
+            style={"display": "none"},
+            children=[
+                dcc.Link(dcc.Markdown("Opportunities"), href="/opportunities"),
+                dcc.Link(dcc.Markdown("Leads"), href="/leads"),
+                dcc.Link(dcc.Markdown("Cases"), href="/cases"),
+            ],
+        ),
+        dcc.Store(  # opportunities df
+            id="opportunities_df",
+            data=sf_manager.get_opportunities().to_json(orient="split"),
+        ),
+        dcc.Store(  # leads df
+            id="leads_df", data=sf_manager.get_leads().to_json(orient="split")
+        ),
+        dcc.Store(
+            id="cases_df", data=sf_manager.get_cases().to_json(orient="split")
+        ),  # cases df
+        dcc.Location(id="url", refresh=False),
+        html.Div(id="tab_content"),
+        html.Link(
+            href="https://use.fontawesome.com/releases/v5.2.0/css/all.css",
+            rel="stylesheet",
+        ),
+        html.Link(
+            href="https://fonts.googleapis.com/css?family=Dosis", rel="stylesheet"
+        ),
+        html.Link(
+            href="https://fonts.googleapis.com/css?family=Open+Sans", rel="stylesheet"
+        ),
+        html.Link(
+            href="https://fonts.googleapis.com/css?family=Ubuntu", rel="stylesheet"
+        ),
+    ],
+    className="row",
+    style={"margin": "0%"},
+)
+
+# Update the index
+
+
+@app.callback(
+    [
+        dash.dependencies.Output("tab_content", "children"),
+        dash.dependencies.Output("tabs", "children"),
+        dash.dependencies.Output("mobile_tabs", "children"),
```

Comment:
I think to be consistent with the other files. Just use `Input`, `Output` and import the dash dependencies at the top.